### PR TITLE
Removed unknown numbers

### PR DESCRIPTION
**Pull request by JasonBrown111**
Removed unknown numbers in points 17 and 19 of "Incremental improvement of the policy statements"

### CODE REVIEW

**Comment by PRMerger13** [comment]
@JasonBrown111 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/govern/guides/complex/security-baseline-improvement.md [hunk 1]
```diff
@@ -84,9 +84,9 @@ The following changes to policy will help remediate the new risks and guide impl
...
+1. Deployment tooling must be approved by the cloud governance team to ensure ongoing governance of deployed assets. Deployment scripts must be maintained in a central repository accessible by the cloud governance team for periodic review and auditing.
 1. Governance processes must include audits at the point of deployment and at regular cycles to ensure consistency across all assets.
-1. Deployment of any applications that require customer authentication must use an approved identity provider that is compatible with the primary identity provider for internal users. 1. Cloud governance processes must include quarterly reviews with identity baseline teams to identify malicious actors or usage patterns that should be prevented by cloud asset configuration.
+1. Deployment of any applications that require customer authentication must use an approved identity provider that is compatible with the primary identity provider for internal users. Cloud governance processes must include quarterly reviews with identity baseline teams to identify malicious actors or usage patterns that should be prevented by cloud asset configuration.
```
Comment (ShannonLeavitt) [inline docs/govern/guides/complex/security-baseline-improvement.md:89]:
```suggestion
1. Deployment of any applications that require customer authentication must use an approved identity provider that is compatible with the primary identity provider for internal users. 
1. Cloud governance processes must include quarterly reviews with identity baseline teams to identify malicious actors or usage patterns that should be prevented by cloud asset configuration.
```
Fixing formatting

File: docs/govern/guides/complex/security-baseline-improvement.md [hunk 2]
```diff
@@ -84,9 +84,9 @@ The following changes to policy will help remediate the new risks and guide impl
 1. The cloud vendor must be capable of supporting the existing edge device solution and any required configurations to protect any publicly exposed network boundary.
 1. The cloud vendor must be capable of supporting a shared connection to the global WAN, with data transmission routed through the existing edge device solution.
 1. Trends and exploits that could affect cloud deployments should be reviewed regularly by the security team to provide updates to Security Baseline tools used in the cloud.
-1. Deployment tooling must be approved by the cloud governance team to ensure ongoing governance of deployed assets. 18. Deployment scripts must be maintained in a central repository accessible by the cloud governance team for periodic review and auditing.
+1. Deployment tooling must be approved by the cloud governance team to ensure ongoing governance of deployed assets. Deployment scripts must be maintained in a central repository accessible by the cloud governance team for periodic review and auditing.
```
Comment (ShannonLeavitt) [inline docs/govern/guides/complex/security-baseline-improvement.md:87]:
```suggestion
1. Deployment tooling must be approved by the cloud governance team to ensure ongoing governance of deployed assets. 
1. Deployment scripts must be maintained in a central repository accessible by the cloud governance team for periodic review and auditing.
```
Fixing formatting

**Comment by ShannonLeavitt** [comment]
Thanks for pointing out the problem @JasonBrown111 

I've made a small adjustment and will merge shortly.

**Pull request merged**